vacasa/connect/connect.py

```diff
@@ -632,13 +633,18 @@ def create_reservation(self,
         if cvv:
             payload['cvv'] = str(cvv)
 
-        # The discount_id is optional. Add it if we've got it.
         if discount_id:
             payload['discount_id'] = discount_id
 
         if initial_payment_amount:
             payload['initial_payment_amount'] = initial_payment_amount
 
+        if anonymous_id is not None:
+            if _is_uuid4(anonymous_id):
+                payload['anonymous_id'] = anonymous_id
+            else:
+                logger.warning("Ignoring invalid UUID4: %s", anonymous_id)
```

Comment:
Doesn't matter but I'd use an f-string instead.

Reply:
This is one of those things where it's an insignificant micro-optimization and I should probably stop doing it, but it's because the logger will actually defer the string interpolation until after it evaluates the loglevel (so you don't do the string interpolation if you're doing a `logger.debug` but have the level set to `WARNING`).  But yeah... totally not necessary here

https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles

Reply:
Interesting. I didn't know that.